closeio/contrib/testing_stub.py

```diff
@@ -689,3 +689,60 @@ def api_key(self):
                 'key': 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx',
             }],
         })
+
+    @parse_response
+    def get_webhooks(self):
+        return self._data('webhooks', [])
+
+    @parse_response
+    def get_webhook(self, webhook_id):
+        webhooks = self._data('webhooks', [])
+
+        for webhook in webhooks:
+            if webhook['id'] == webhook_id:
+                return webhook
+
+        raise CloseIOError()
+
+    @parse_response
+    def create_webhook(self, data):
+        webhooks = self._data('webhooks', [])
+
+        new_webhook = {
+            'status': 'active',
+            'updated_by': 'user_{}'.format(uuid.uuid4().hex),
+            'url': data['url'],
+            'created_by': 'user_{}'.format(uuid.uuid4().hex),
+            'signature_key': 'xxxxxx',
+            'date_created': datetime.utcnow().isoformat(),
+            'date_updated': datetime.utcnow().isoformat(),
+            'verify_ssl': True,
+            'events': data['events'],
+            'id': 'whsub_{}'.format(uuid.uuid4().hex)
+        }
+        webhooks.append(new_webhook)
+        return new_webhook
+
+    @parse_response
+    def update_webhook(self, webhook_id, status):
+        webhook = self.get_webhook(webhook_id)
+        if webhook:
+            webhook['status'] = status['status']
+            return webhook
+
+        raise CloseIOError()
+
+    @parse_response
+    def delete_webhook(self, webhook_id):
+        webhooks = self._data('webhooks', [])
+        found = -1
+
+        for index, webhook in enumerate(webhooks):
+            if webhook['id'] == webhook_id:
+                found = index
+
+        if found == -1:
+            raise CloseIOError()
+
+        webhooks.pop(found)
+        return True
```

Comment:
this definitely works. I personally would probably prefer to write this with a `for...else` loop: 
```
webhooks = self._data('webhooks', [])
for index, webhook in enumerate(webhooks):
    if webhook['id'] == webhook_id:
        webhooks.pop(index)
        return True
else:
    raise CloseIOError()
```
This approach has the advantage that you stop iterating over all hooks once you've found the right one. And it's a bit less code.

But it's a style question. You can take this comment only as a suggestion.

Reply:
A really nice suggestion. Yep, it works fine and makes the code cleaner. 👍